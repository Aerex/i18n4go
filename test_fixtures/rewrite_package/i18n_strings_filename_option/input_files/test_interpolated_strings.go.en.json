[
   {
      "id": "Hello %s world!",
      "translation": "Hello %s world!"
   },
   {
      "id": "Hello %d(%s) world!, bye from %s",
      "translation": "Hello %d(%s) world!, bye from %s"
   },
   {
      "id": "Evil",
      "translation": "Evil"
   }
]
