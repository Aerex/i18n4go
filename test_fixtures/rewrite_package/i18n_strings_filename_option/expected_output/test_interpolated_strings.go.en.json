[
   {
      "id": "Hello {{.Arg0}}({{.Arg1}}) world!, bye from {{.Arg2}}",
      "translation": "Hello {{.Arg0}}({{.Arg1}}) world!, bye from {{.Arg2}}"
   },
   {
      "id": "Evil",
      "translation": "Evil"
   },
   {
      "id": "Hello {{.Arg0}} world!",
      "translation": "Hello {{.Arg0}} world!"
   }
]
